app/javascript/utils/form/final_form/Field.js

```diff
@@ -220,7 +265,7 @@ export const TextCheckboxField = ({ fieldName, label, blockNote, validation, id,
             meta={meta}
             aria-labelledby={ariaLabelledby}
             id={id || `form-base-${fieldName}`} />
-          <input id={id || `form-ctrl-${fieldName}`} fieldName={fieldName} type='checkbox' checked={input.value === trueValue} onClick={onChange} />
```

Comment:
Why remove the fieldName prop here?

Reply:
it was actually causing console log errors! not sure where it came from or when they started showing up